cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/KafkaRebalanceStateMachineTest.java

```diff
@@ -96,12 +97,13 @@ public void resetServer() {
      * @param currentStatusSessionID The user task ID attached to the current KafkaRebalance resource. Can be null.
      * @param userAnnotation An annotation to be applied after the reconcile has started, for example "approve" or "stop".
      * @param rebalanceSpec A custom rebalance specification. If null a blank spec will be used.
+     * @param exceptionString
      * @return A KafkaRebalance instance configured with the supplied parameters.
      */
     private KafkaRebalance createKafkaRebalance(KafkaRebalanceState currentState,
                                                 String currentStatusSessionID,
                                                 String userAnnotation,
-                                                KafkaRebalanceSpec rebalanceSpec) {
+                                                KafkaRebalanceSpec rebalanceSpec, String exceptionString) {
```

Comment:
I see it sets the reason on the condition, why not calling it `reason` instead of `exceptionString`?